backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/invoice/isCompleteInvoice.feature

```diff
@@ -0,0 +1,120 @@
+@from:cucumber
+@dev:runThisOne
+Feature: completeInvoice option when processing invoice candidates
+
+  Background:
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And set sys config boolean value true for sys config SKIP_WP_PROCESSOR_FOR_AUTOMATION
+    And set sys config boolean value true for sys config de.metas.report.jasper.IsMockReportService
+    And metasfresh has date and time 2021-12-21T13:30:13+01:00[Europe/Berlin]
```

Comment:
why do you need to set the time? 
like with the API token, if there i no need, better don't do it to avoid FUD

Reply:
without it another feature may set the date before the order date, which results in order can't be invoiced